2013/11/22 03:38:38 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/auth/CAccessControlFilter.php:182
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/auth/CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(291): CFilterChain->run()
#6 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(282): CController->run('admin')
#8 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('department/admi...')
#9 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#10 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/~Adi/www/test0/index.php/department/admin
HTTP_REFERER=http://localhost/~Adi/www/test0/index.php/department/create
---
2013/11/22 03:39:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "sdf".' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php:286
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('sdf')
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/~Adi/www/test0/index.php/sdf
---
2013/11/22 03:59:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "abou".' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php:483
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(270): CController->missingAction('abou')
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(282): CController->run('abou')
#2 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('site/abou')
#3 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/~Adi/www/test0/index.php/site/abou
---
2013/11/22 03:59:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "about".' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php:483
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(270): CController->missingAction('about')
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(282): CController->run('about')
#2 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('site/about')
#3 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/~Adi/www/test0/index.php/site/about
---
2013/11/22 03:59:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "abou".' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php:483
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(270): CController->missingAction('abou')
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(282): CController->run('abou')
#2 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('site/abou')
#3 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/~Adi/www/test0/index.php/site/abou
---
2013/11/22 04:04:23 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '10/22/1999' for column 'hireDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6).
in /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php (73)
in /Users/Adi/Sites/www/test0/index.php (13)
2013/11/22 04:04:23 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '10/22/1999' for column 'hireDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6)' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/CDbCommand.php:358
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(1077): CDbCommand->execute()
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php(73): CActiveRecord->save()
#3 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/actions/CInlineAction.php(49): EmployeeController->actionCreate()
#4 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(291): CFilterChain->run()
#11 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(282): CController->run('create')
#13 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('employee/create')
#14 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/~Adi/www/test0/index.php/employee/create
HTTP_REFERER=http://localhost/~Adi/www/test0/index.php/employee/create
---
2013/11/22 04:06:57 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '10/22/1999' for column 'hireDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6).
in /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php (73)
in /Users/Adi/Sites/www/test0/index.php (13)
2013/11/22 04:06:57 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '10/22/1999' for column 'hireDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6)' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/CDbCommand.php:358
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(1077): CDbCommand->execute()
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php(73): CActiveRecord->save()
#3 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/actions/CInlineAction.php(49): EmployeeController->actionCreate()
#4 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(291): CFilterChain->run()
#11 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(282): CController->run('create')
#13 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('employee/create')
#14 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/~Adi/www/test0/index.php/employee/create
HTTP_REFERER=http://localhost/~Adi/www/test0/index.php/employee/create
---
2013/11/22 05:01:19 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'leaveDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6).
in /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php (73)
in /Users/Adi/Sites/www/test0/index.php (13)
2013/11/22 05:01:19 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'leaveDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6)' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/CDbCommand.php:358
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(1077): CDbCommand->execute()
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php(73): CActiveRecord->save()
#3 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/actions/CInlineAction.php(49): EmployeeController->actionCreate()
#4 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(291): CFilterChain->run()
#11 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(282): CController->run('create')
#13 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('employee/create')
#14 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/~Adi/www/test0/index.php/employee/create
HTTP_REFERER=http://localhost/~Adi/www/test0/index.php/employee/create
---
2013/11/22 05:01:41 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'leaveDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6).
in /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php (73)
in /Users/Adi/Sites/www/test0/index.php (13)
2013/11/22 05:01:41 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'leaveDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6)' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/CDbCommand.php:358
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(1077): CDbCommand->execute()
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php(73): CActiveRecord->save()
#3 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/actions/CInlineAction.php(49): EmployeeController->actionCreate()
#4 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(291): CFilterChain->run()
#11 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(282): CController->run('create')
#13 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('employee/create')
#14 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/~Adi/www/test0/index.php/employee/create
HTTP_REFERER=http://localhost/~Adi/www/test0/index.php/employee/create
---
2013/11/22 05:07:45 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'leaveDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6).
in /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php (73)
in /Users/Adi/Sites/www/test0/index.php (13)
2013/11/22 05:07:45 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'leaveDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6)' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/CDbCommand.php:358
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(1077): CDbCommand->execute()
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php(73): CActiveRecord->save()
#3 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/actions/CInlineAction.php(49): EmployeeController->actionCreate()
#4 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(291): CFilterChain->run()
#11 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(282): CController->run('create')
#13 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('employee/create')
#14 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/~Adi/www/test0/index.php/employee/create
HTTP_REFERER=http://localhost/~Adi/www/test0/index.php/employee/create
---
2013/11/22 05:42:11 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'leaveDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6).
in /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php (73)
in /Users/Adi/Sites/www/test0/index.php (13)
2013/11/22 05:42:11 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'leaveDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6)' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/CDbCommand.php:358
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(1077): CDbCommand->execute()
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php(73): CActiveRecord->save()
#3 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/actions/CInlineAction.php(49): EmployeeController->actionCreate()
#4 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(291): CFilterChain->run()
#11 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(282): CController->run('create')
#13 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('employee/create')
#14 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/~Adi/www/test0/index.php/employee/create
HTTP_REFERER=http://localhost/~Adi/www/test0/index.php/employee/create
---
2013/11/22 05:43:26 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'leaveDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6).
in /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php (73)
in /Users/Adi/Sites/www/test0/index.php (13)
2013/11/22 05:43:26 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'leaveDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6)' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/CDbCommand.php:358
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(1077): CDbCommand->execute()
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php(73): CActiveRecord->save()
#3 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/actions/CInlineAction.php(49): EmployeeController->actionCreate()
#4 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(291): CFilterChain->run()
#11 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(282): CController->run('create')
#13 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('employee/create')
#14 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/~Adi/www/test0/index.php/employee/create
HTTP_REFERER=http://localhost/~Adi/www/test0/index.php/employee/create
---
2013/11/22 05:44:44 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '2012-09-13T17:58:23+02:00' for column 'hireDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6).
in /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php (73)
in /Users/Adi/Sites/www/test0/index.php (13)
2013/11/22 05:44:44 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '2012-09-13T17:58:23+02:00' for column 'hireDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6)' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/CDbCommand.php:358
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(1077): CDbCommand->execute()
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php(73): CActiveRecord->save()
#3 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/actions/CInlineAction.php(49): EmployeeController->actionCreate()
#4 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(291): CFilterChain->run()
#11 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(282): CController->run('create')
#13 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('employee/create')
#14 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/~Adi/www/test0/index.php/employee/create
HTTP_REFERER=http://localhost/~Adi/www/test0/index.php/employee/create
---
2013/11/22 05:48:04 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'leaveDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6).
in /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php (73)
in /Users/Adi/Sites/www/test0/index.php (13)
2013/11/22 05:48:04 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'leaveDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6)' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/CDbCommand.php:358
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(1077): CDbCommand->execute()
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php(73): CActiveRecord->save()
#3 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/actions/CInlineAction.php(49): EmployeeController->actionCreate()
#4 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(291): CFilterChain->run()
#11 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(282): CController->run('create')
#13 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('employee/create')
#14 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/~Adi/www/test0/index.php/employee/create
HTTP_REFERER=http://localhost/~Adi/www/test0/index.php/employee/create
---
2013/11/22 05:49:19 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '1299762201428' for column 'hireDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6).
in /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php (73)
in /Users/Adi/Sites/www/test0/index.php (13)
2013/11/22 05:49:19 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '1299762201428' for column 'hireDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6)' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/CDbCommand.php:358
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(1077): CDbCommand->execute()
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php(73): CActiveRecord->save()
#3 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/actions/CInlineAction.php(49): EmployeeController->actionCreate()
#4 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(291): CFilterChain->run()
#11 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(282): CController->run('create')
#13 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('employee/create')
#14 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/~Adi/www/test0/index.php/employee/create
HTTP_REFERER=http://localhost/~Adi/www/test0/index.php/employee/create
---
2013/11/22 05:58:26 [warning] [application] Failed to set unsafe attribute "hireDate" of "Employee".
in /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php (72)
in /Users/Adi/Sites/www/test0/index.php (13)
2013/11/22 05:58:26 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'leaveDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5).
in /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php (73)
in /Users/Adi/Sites/www/test0/index.php (13)
2013/11/22 05:58:26 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'leaveDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5)' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/CDbCommand.php:358
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(1077): CDbCommand->execute()
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php(73): CActiveRecord->save()
#3 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/actions/CInlineAction.php(49): EmployeeController->actionCreate()
#4 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(291): CFilterChain->run()
#11 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(282): CController->run('create')
#13 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('employee/create')
#14 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/~Adi/www/test0/index.php/employee/create
HTTP_REFERER=http://localhost/~Adi/www/test0/index.php/employee/create
---
2013/11/22 05:58:41 [warning] [application] Failed to set unsafe attribute "hireDate" of "Employee".
in /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php (72)
in /Users/Adi/Sites/www/test0/index.php (13)
2013/11/22 05:58:41 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'leaveDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5).
in /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php (73)
in /Users/Adi/Sites/www/test0/index.php (13)
2013/11/22 05:58:41 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'leaveDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5)' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/CDbCommand.php:358
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(1077): CDbCommand->execute()
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php(73): CActiveRecord->save()
#3 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/actions/CInlineAction.php(49): EmployeeController->actionCreate()
#4 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(291): CFilterChain->run()
#11 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(282): CController->run('create')
#13 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('employee/create')
#14 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/~Adi/www/test0/index.php/employee/create
HTTP_REFERER=http://localhost/~Adi/www/test0/index.php/employee/create
---
2013/11/22 06:01:44 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'hireDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6).
in /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php (73)
in /Users/Adi/Sites/www/test0/index.php (13)
2013/11/22 06:01:44 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '' for column 'hireDate' at row 1. The SQL statement executed was: INSERT INTO `Employee` (`departmentId`, `firstName`, `lastName`, `email`, `ext`, `hireDate`, `leaveDate`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6)' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/CDbCommand.php:358
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(1077): CDbCommand->execute()
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/db/ar/CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 /Users/Adi/Sites/www/test0/protected/controllers/EmployeeController.php(73): CActiveRecord->save()
#3 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/actions/CInlineAction.php(49): EmployeeController->actionCreate()
#4 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(291): CFilterChain->run()
#11 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(282): CController->run('create')
#13 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('employee/create')
#14 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/~Adi/www/test0/index.php/employee/create
HTTP_REFERER=http://localhost/~Adi/www/test0/index.php/employee/create
---
2013/11/23 06:06:57 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/auth/CAccessControlFilter.php:182
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/auth/CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(291): CFilterChain->run()
#6 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(282): CController->run('admin')
#8 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('employee/admin')
#9 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#10 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/~Adi/www/test0/index.php/employee/admin
HTTP_REFERER=http://localhost/~Adi/www/test0/index.php/employee/index
---
2013/12/03 15:17:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Users".' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php:286
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('Users')
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/~Adi/www/test0/index.php/Users
---
2013/12/03 15:17:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "users".' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php:286
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('users')
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/~Adi/www/test0/index.php/users
---
2013/12/03 15:17:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user".' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php:286
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('user')
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/~Adi/www/test0/index.php/user
---
2013/12/03 16:41:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "users".' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php:483
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CController.php(270): CController->missingAction('users')
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(282): CController->run('users')
#2 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('site/users')
#3 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/~Adi/www/test0/index.php/site/users
---
2013/12/03 16:43:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "views/users".' in /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php:286
Stack trace:
#0 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/web/CWebApplication.php(141): CWebApplication->runController('views/users')
#1 /Users/Adi/Sites/yii-1.1.14.f0fee9/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Users/Adi/Sites/www/test0/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/~Adi/www/test0/index.php/views/users
---
